rclcpp/test/rclcpp/test_timers_manager.cpp

```diff
@@ -296,12 +237,11 @@ TEST_F(TestTimersManager, timers_thread)
 
   // Run timers thread for a while
   timers_manager->start();
-  std::this_thread::sleep_for(5ms);
+  std::this_thread::sleep_for(50ms);
   timers_manager->stop();
 
   EXPECT_LT(1u, t1_runs);
   EXPECT_LT(1u, t2_runs);
-  EXPECT_EQ(t1_runs, t2_runs);
```

Comment:
what about
```
EXPECT_LE(t1_runs - t2_runs, 1u);
```

Reply:
Yeah, that's fair.  Fixed in f9de6eb6810e69f96e64672c3267ac7ee9360165 (note I had to change the types of `t1_runs` and `t2_runs`, because they now need to be able to be promoted to signed integers for the subtraction).